### PR TITLE
horizontal-scroll-bug-on-some-pages | [Feels hacky, not merging it yet]

### PR DESCRIPTION
**Pull request by ntorbinskiy**
code blocks and removed horizontal scroll bars

### CODE REVIEW

File: styles/main.scss [hunk 1]
```diff
@@ -884,7 +884,7 @@ h2 {
   -ms-flex-positive: 1;
   flex-grow: 1;
   border-right: 1px solid #e7e7e7;
-  padding: 36px 30px;
+  padding: 36px 20px;
```
Comment (alexander-vashchuk) [inline styles/main.scss:887]: @ntorbinskiy you are likely should not modify those kinds of global styles

File: styles/responsive.scss [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+    max-width: 48vw;
+  }
+}
+
+@media screen and (max-width: 1180px) and (min-width: 1021px) {
+  .remark-highlight {
+    max-width: 59vw;
+  }
+}
+
```
Comment (alexander-vashchuk) [inline styles/responsive.scss:31]: Um, feels very hacky as for me

Comment (ntorbinskiy) [reply styles/responsive.scss:31]: What's wrong?

Comment (ntorbinskiy) [reply styles/responsive.scss:31]: @alexander-vashchuk @1alexvash 
What's wrong?

Comment (alexander-vashchuk) [reply styles/responsive.scss:31]: Feels hacky, we don't have those breakpoints in the project

Comment (alexander-vashchuk) [reply styles/responsive.scss:31]: Maybe you should try calc property to limit the width

**Pull request closed without merging**